### ISSUE TITLE
Require the minimum length of 8 characters for wpa key entry.

### ISSUE THREAD

**Issue by Vigilence** [sophron/wifiphisher#203]
It adds legitimacy to the page for those that my try to bypass the page by entering something random.

**Comment by blackHatMonkey** [comment]
It's a very nice idea and easy to implement. However one down side to this is if the encryption type for the ```ESSID``` is ```WEP``` which is either 5 or 13 characters.

**Comment by Vigilence** [comment]
I hadn't thought about WEP.  But if someone is eager to get a WEP key that can easily be obtained through an automated process using wifite on Kali Linux.

Depending on distance and the amount of clients connected a WEP key can easily be obtained in a matter of minutes.

**Comment by sophron** [comment]
@Vigilence Thanks, we'll add this.

@blackHatMonkey The script can determine if the target network is WPA or WEP encrypted, hence it can dynamically adjust this value.

**Comment by Vigilence** [comment]
Wow nice.  Have the script adjust its parameters based on the encryption would be super cool.

Thanks.

**Comment by laozi999** [comment]
This can be achieved using javascript:

```
$('form').submit(function(e){
    e.preventDefault();

    if ( $('input[type=password]').val().length < 8 ) {
        // show error or something
    } else {
        $.post( '/post', 'wfphshr-wpa-password=' + $('input[type=password]').val() );
        // success, do something
    }
});
```

**Comment by sophron** [comment]
@laozi999 Yes. Feel free to make a PR on this if you like. You can see my complete view on this issue on #290.

**Comment by sophron** [comment]
@laozi999 Yes. Feel free to make a PR on this if you like.

**Comment by laozi999** [comment]
Should it always be implemented though? Since some APs can have a WEP key of 5 characters.

**Comment by laozi999** [comment]
I guess wifiphisher wouldn't necessarily be the best option for WEP keys, but it sure will work.
